parse.py

```diff
@@ -1,3 +1,14 @@
+# -*- coding: UTF-8 -*-
+# BASED-ON: https://github.com/r1chardj0n3s/parse/parse.py
+# VERSION:  parse 1.8.2
+# Same as original parse module except the following extensions.
+# EXTENSIONS:
+#  * group_count attribute support for user-defined type converters
+#     => FIXES: group_index offset problem for fixed fields
+#        when pattern has groups
+#  * raise StopIteration() instead of StopIteration (as class)
+#
+#  -- ORIGINAL-CODE STARTS-HERE ------------------------------------------------
```

Comment:
I don't think we need to include the above note in this patch.

Reply:
Correct.
That was informational to understand what changes are made.
I remove it.